e2e/test/iothub/service/RegistryE2ETests.cs

```diff
@@ -353,7 +353,7 @@ public async Task RegistryManager_ConfigurationOperations_Work()
 
             bool configCreated = false;
             string configurationId = (_idPrefix + Guid.NewGuid()).ToLower(); // Configuration Id characters must be all lower-case.
-            using var client = RegistryManager.CreateFromConnectionString(TestConfiguration.IoTHub.ConnectionString);
+            using var sc = new IotHubServiceClient(TestConfiguration.IoTHub.ConnectionString);
```

Comment:
This test method only tests the ConfigurationsClient, so you should move this to a new test class called "ConfigurationsClientE2ETests"

Reply:
added the relevant changes